src/meltano/core/project_add_service.py

```diff
@@ -106,9 +112,12 @@ def add_related(self, *args, **kwargs):
         Returns:
             The added plugins.
         """
+        if project_plugin.requirements:
+            return self.add_required(project_plugin)
```

Comment:
@edgarrmondragon Now that all plugins on the hub have `requires` (https://github.com/meltano/hub/pull/538), do you think we can remove the whole discovery/namespace-based `add_related` code path? I added this in with a conditional just to be sure.